### ISSUE TITLE
false positive with no-namespace allow-declarations

### ISSUE THREAD

**Issue by jpzwarte** [palantir/tslint#1833]
### Bug Report

- __TSLint version__: `4.0.2`
- __TypeScript version__: `2.1.4`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
declare global {
  interface Document {
     // ...
  }
}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "no-namespace": [
      true,
      "allow-declarations"
    ]
  }
}
```

#### Actual behavior

```
foo.ts[5, 1]: 'namespace' and 'module' are disallowed
error Command failed with exit code 2.
```

#### Expected behavior

No error.

**Comment by andy-hanson** [comment]
Actually, it looks like this is already fixed in `tslint@4.1.0-dev.0`.
